.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Configure LF line endings for text files
```

Comment:
Ah, it's been a while since working on this – I might have messed it up from a different PR and added it here instead. I think it was for SciPy's BLAS signatures text file. It's nice to include to ensure we have consistent line endings across platforms, but I'll do this in a separate PR. Removed in ce7e6ccfaf29e591b720bc62d49b302f933a6e6f.